src/services/doppler-contact-policy-api-client.ts

```diff
@@ -107,7 +107,7 @@ export class HttpDopplerContactPolicyApiClient implements DopplerContactPolicyAp
             : [],
           timeRestriction: response.data.timeRestriction
             ? this.mapTimeRestriction(response.data.timeRestriction)
-            : null,
+            : this.mapTimeRestriction({}),
```

Comment:
Almost the same, but, in my opinion, it is simpler and makes it easier to understand why you are using an empty object.

```suggestion
          timeRestriction: this.mapTimeRestriction(response.data.timeRestriction ?? {}),
```

One alternative could be to accept falsy values as the parameter of `mapTimeRestriction`

Reply:
Yes, I am going to fix it.

Reply:
Done: https://github.com/FromDoppler/doppler-webapp/pull/2385/commits/ef6304825bd264cada7c8f5fb327154a7e97c694